packages/js/product-editor/src/blocks/product-fields/inventory-sku/edit.tsx

```diff
@@ -33,11 +33,8 @@ export function Edit( {
 }: ProductEditorBlockEditProps< BlockAttributes > ) {
 	const blockProps = useWooBlockProps( attributes );
 
-	const [ sku, setSku ] = useEntityProp(
-		'postType',
-		context.postType,
-		'sku'
-	);
+	const [ sku, setSku ]: [ string, ( newSku: string ) => void, unknown ] =
+		useEntityProp( 'postType', context.postType, 'sku' );
```

Comment:
```suggestion
	const [ sku, setSku ] =
		useEntityProp<string>( 'postType', context.postType, 'sku' );
```
`useEntityProp` accepts a generic type.